example/gardener-local/etcd/kustomization.yaml

```diff
@@ -5,8 +5,6 @@ generatorOptions:
   disableNameSuffixHash: true
 
 namespace: garden
-commonLabels:
-  app: etcd
```

Comment:
It was this way because of several challenges I faced with the skaffold configuration. Anyways, I have found a way now to revert this, and I think that this will also enable the upgrade tests to run. Thanks for pointing it out!